src/test/java/bio/terra/service/rawls/RawlsServiceTest.java

```diff
@@ -22,97 +23,67 @@
 
-@RunWith(SpringRunner.class)
-@SpringBootTest(properties = {"datarepo.testWithEmbeddedDatabase=false"})
-@AutoConfigureMockMvc
 @ActiveProfiles({"google", "unittest"})
 @Category(Unit.class)
+@RunWith(MockitoJUnitRunner.StrictStubs.class)
 public class RawlsServiceTest {
 
-  @Mock private SamConfiguration samConfig;
-  @Mock private ConfigurationService configurationService;
-  @Mock private ApiClient apiClient;
-  @Mock private ResourcesApi samResourceApi;
-  @Mock private StatusApi samStatusApi;
-  @Mock private GoogleApi samGoogleApi;
-  @Mock private UsersApi samUsersApi;
-
-  @Mock private AuthenticatedUserRequest userReq;
-
-  @MockBean private RawlsClient rawlsClient;
-  @Autowired private RawlsService rawlsService;
-  private SamIam samIam;
+  @Mock private TerraConfiguration terraConfiguration;
+  @Mock private RawlsClient rawlsClient;
+  private RawlsService rawlsService;
+  private static final AuthenticatedUserRequest TEST_USER =
+      AuthenticationFixtures.randomUserRequest();
+  private static final String BASE_PATH = "terra.base.path";
   private static final String OWNER_NAME = "owner";
   private static final String OWNER_EMAIL = "policy-email@firecloud.org";
   private static final String WORKSPACE_NAME = "testWorkspace";
-  private static final String WORKSPACE_NAMEPACE = "testNamespace";
+  private static final String WORKSPACE_NAMESPACE = "testNamespace";
   private final UUID accessibleWorkspaceId = UUID.randomUUID();
+  private final WorkspaceResponse accessibleWorkspaceResponse =
+      new WorkspaceResponse()
+          .workspace(
+              new WorkspaceDetails()
+                  .workspaceId(accessibleWorkspaceId.toString())
+                  .name(WORKSPACE_NAME)
+                  .namespace(WORKSPACE_NAMESPACE));
   private final UUID inaccessibleWorkspaceId = UUID.randomUUID();
   private final UnauthorizedException inaccessibleWorkspaceException =
       new UnauthorizedException("Workspace inaccessible");
 
   @Before
   public void setUp() throws Exception {
-    MockitoAnnotations.initMocks(this);
-    samIam = spy(new SamIam(samConfig, configurationService));
-    final String userToken = "some_token";
-    when(userReq.getToken()).thenReturn(userToken);
-    when(configurationService.getParameterValue(ConfigEnum.SAM_RETRY_MAXIMUM_WAIT_SECONDS))
-        .thenReturn(0);
-    when(configurationService.getParameterValue(ConfigEnum.SAM_RETRY_INITIAL_WAIT_SECONDS))
-        .thenReturn(0);
-    when(configurationService.getParameterValue(ConfigEnum.SAM_OPERATION_TIMEOUT_SECONDS))
-        .thenReturn(0);
-    // Mock out samApi, samStatusApi, samGoogleApi, and samUsersApi in individual tests as needed
-    doAnswer(a -> samResourceApi).when(samIam).samResourcesApi(userToken);
-    // Mock out the lower level client in individual as needed
-    when(samResourceApi.getApiClient()).thenAnswer(a -> apiClient);
+    rawlsService = new RawlsService(terraConfiguration, rawlsClient);
```

Comment:
Nice, this is much better than `@Autowired` since it gives you explicit control over the object creation.